src/mongocxx/options/find.hpp

```diff
@@ -455,6 +455,16 @@ class MONGOCXX_API find {
     ///
     const stdx::optional<bsoncxx::document::view_or_value>& sort() const;
 
+    ///
+    /// Modifies the object to convert all modifiers into their respective top-level options. For
+    /// internal use only.
+    ///
+    /// @throws
+    ///   mongocxx::exception if any modifiers are invalid, or if the unsupported "$query" or
```

Comment:
I've decided here to break applications that use `$query` or `$explain` in modifiers.  Is this acceptable?